src/Contract Test Environment.postman_environment.json

```diff
@@ -76,9 +77,15 @@
 			"enabled": true
+		},
+		{
+			"key": "env-openapi-json-url",
```

Comment:
this is an option used for the agent protocol:
The agent protocol is a public API spec. So we just want to be able to use the schema in github.